<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="anonymousFindFaces">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>390</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Finn ansikter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="anonymousOriginal">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
  </widget>
  <widget class="QLabel" name="glattingOverskrift">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>761</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Anonymisering av ansikter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>201</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Finner ansiktene i bildet, og&lt;br/&gt;anonymiserer disse ansiktene&lt;br/&gt;ved å glatte disse områdene.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Velg mellom utvalg av bilder&lt;br/&gt;med ansikter til høyre. Se &lt;br/&gt;originalversjonen av bildet,&lt;br/&gt;finn ansiktene eller anonymiser&lt;br/&gt;disse ved hjelp av knappene &lt;br/&gt;nederst. Spinboxen til venstre &lt;br/&gt;endrer scaleFactor, mens &lt;br/&gt;spinboxen til høyre endrer&lt;br/&gt;minNeighbours. Nede til høyre &lt;br/&gt;står det hvor mange ansikter &lt;br/&gt;som er oppdaget. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Lagre bildet ved å klikke på&lt;br/&gt;lagreknappen nederst til høyre.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="group1">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>120</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Group 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="couple">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Couple</string>
   </property>
  </widget>
  <widget class="QPushButton" name="anonymousCode">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vis kode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="anonymousAnonymise">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>390</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Anonymiser</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>360</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ansikt:</string>
   </property>
  </widget>
  <widget class="QLabel" name="faceCount">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>360</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="group2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>160</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Group 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="group3">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>200</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Group 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="family">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>240</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Family</string>
   </property>
  </widget>
  <widget class="QPushButton" name="business">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>280</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Business</string>
   </property>
  </widget>
  <widget class="QPushButton" name="music">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>320</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Music</string>
   </property>
  </widget>
  <widget class="imagewidget" name="imagewidget" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>79</y>
     <width>471</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>390</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lagre</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="scaleFactor">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>390</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="statusTip">
    <string extracomment="ScaleFactor"/>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="maximum">
    <double>4.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.002000000000000</double>
   </property>
   <property name="value">
    <double>1.200000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="minNeighbours">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>390</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string extracomment="MinNeighbours"/>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>imagewidget</class>
   <extends>QWidget</extends>
   <header>imagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
